agent/app/healthcheck.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2014-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: -2019

Reply:
`Copyright 2019 Amazon.com` since it's a new file?